README.rst

```diff
@@ -1,20 +1,26 @@
-****************
+
+******************************
 django CMS Version Locking
-****************
+******************************
 
-## Explanation
-The version-locking addon is intended to modify the way djangocms-versioning works.  
-* The primary change is that it locks a version to its author when a draft is created. 
-* That version becomes automatically unlocked again once it is published. 
-* The lock prevents editing of the file by anyone other than the author.
-* Locks can be removed by a user with the correct permission
-* Unlocking an item sends a notification to the author to which it was locked.
-* Manually unlocking a version does not lock it to the unlocking user, nor does it change the author.
-* The Version admin view for each content-type registered with Version-locking will be edited to add in lock icons / buttons UI to the Actions column.
+Explanation
+------------
+The version-locking addon is intended to modify the way djangocms-versioning works in the following way: 
 
+ - The primary change is that it locks a version to its author when a draft is created. 
```

Comment:
This might be a tiny bit clearer if you explain what locking means straight away. You don't actually explain that until point 3, so my initial reaction was "but what does locking a version to its author mean?"